app/models/concerns/filterable.rb

```diff
@@ -0,0 +1,13 @@
+module Filterable
+  extend ActiveSupport::Concern
+
+  module ClassMethods
+    def filter(filtering_params)
+      results = self.where(nil)
```

Comment:
@christophermlne as I see in the command line this is equal to `self.all`. I thought on it also but I left it as I found.